consensus/src/block_preparer.rs

```diff
@@ -50,14 +50,14 @@ impl BlockPreparer {
         });
         let start_time = Instant::now();
 
-        let mut block_voters = None;
         let (txns, max_txns_from_block_to_execute) = loop {
             tokio::select! {
                 // Poll the block qc future until a QC is received. Ignore None outcomes.
-                Some(qc) = block_qc_fut.clone(), if block_voters.is_none() => {
-                    block_voters = Some(qc.ledger_info().get_voters_bitvec().clone());
+                Some(qc) = block_qc_fut.clone() => {
```

Comment:
nit: clone is not needed, does it still need to be shared?

Reply:
It's because of the loop, which I don't need now. Removed.